packages/web3/src/errors.ts

```diff
@@ -0,0 +1,7 @@
+import { Web3Error } from 'web3-utils';
+
+export class ContractError extends Web3Error {
+	public constructor(message: string) {
+		super('', message);
```

Comment:
```suggestion
		super('ContractError', message);
```

This is what we decided to do for `web3-eth` errors [e.g.](https://github.com/ChainSafe/web3.js/blob/d53cec1c02f198770f75d07cbce9b011b46f6e98/packages/web3-eth/src/errors.ts#L45)